source/includes/mongodb-compatibility-table-php.rst

```diff
@@ -4,6 +4,7 @@
    :class: compatibility-large
 
    * - PHP Driver Versions
+     - MongoDB 5.2
```

Comment:
Related to @kevinAlbs's comment in https://github.com/mongodb/docs-ecosystem/pull/790/files#r797723414, the upcoming releases of PHPC 1.13 and PHPLIB 1.12 will bump our libmongoc dependency to 1.21.0 and add support for MongoDB 5.1 _and_ 5.2.

So this table looks to be correct, since "ext 1.12 + lib 1.11" only advertised support for MongoDB 5.0.

There's a whole separate argument to be had about some of these server versions not requiring driver changes, which probably doesn't mesh well with these compatibility tables. I believe that was the case with MongoDB 5.1. That's probably something for the docs team to bring up separately with Bernie, Rachelle, et al.